packages/sanity/src/desk/panes/document/statusBar/sparkline/PublishStatus.tsx

```diff
@@ -32,7 +32,7 @@ export function PublishStatus(props: PublishStatusProps) {
         portal
         content={
           <Stack padding={3} space={3}>
```

Comment:
```suggestion
          <Stack padding={2} space={2}>
```

**suggestion:** Really minor, but the padding for these tooltips should be consistent with others. This could arguably be a `<Box>` component too – since we never render more than one child here.

We should definitely consider creating a studio-specific tooltip (with set overrides) if we continue to see these sorts of inconsistencies!